internal/bg/initialize.go

```diff
@@ -0,0 +1,23 @@
+// Background jobs responsible for queueing batch operations (availability checks) or other
+// operations (cleanups etc).
+package bg
+
+import (
+	"context"
+	"time"
+
+	"github.com/RHEnVision/provisioning-backend/internal/ctxval"
+)
+
+// Maximum batch size for each batch send, also an incoming buffered channel size to prevent
+// incoming requests to overload the sender.
+const availabilityStatusBatchSize = 1024
+
+// Initialize starts background goroutines. Use context cancellation to stop them.
+func Initialize(ctx context.Context) {
+	logger := ctxval.Logger(ctx).With().Bool("bg", true).Logger()
```

Comment:
could we use `background` or something even more descriptive? `bg` is quite vague and could be missinterpreted by someone not familiar with the code itself.

Reply:
Sure, I renamed this and the package since you mentioned it. It is a bit too long to my taste, but I am okay.